lib/resources/elasticsearch.rb

```diff
@@ -0,0 +1,140 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Rony Xavier, rx294@nyu.edu
+# author: Aaron Lippold, lippold@gmail.com
+# license: All rights reserved
+
+require 'utils/filter'
+
+module Inspec::Resources
+  class Elasticsearch < Inspec.resource(1)
+    name 'elasticsearch'
+    desc 'See Description below'
+    # Use the Elasticsearch InSpec audit resource to test the Elasticsearch
+    # configuration settings. The data is extract from the live elastic search
+    # cluster using the curl command `curl localhost:9200/_nodes/`. The resource
+    # makes uses of FilterTable and its functionality can be used
+
+    example 'See example below'
```

Comment:
The `example` contents are displayed in the `inspec shell` output... so there would be no example printed. Please uncomment similar to other resources.